plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -225,6 +226,10 @@ func (s *Plugin) Validate(ctx context.Context, a admission.Attributes, o admissi
 	if err != nil {
 		return admission.NewForbidden(a, fmt.Errorf("error looking up service account %s/%s: %v", a.GetNamespace(), pod.Spec.ServiceAccountName, err))
 	}
+	// Warn about deprecated annotation
```

Comment:
warning the pod creator about this is weird, *especially* outside the `enforceMountableSecrets` block

Reply:
Do you want to limit it to just the SA create/update?

Reply:
yeah, I think so

Reply:
updated. PTAL